src/calculation/ClauseResultsBuilder.ts

```diff
@@ -363,100 +372,215 @@ export function prettyResult(result: any | null, indentLevel?: number, keyIndent
     let quantityResult = `QUANTITY: ${result.value}`;
     if (result.unit) {
       quantityResult += ` ${result.unit}`;
     }
     return quantityResult;
-  } else if (result && typeof result._type === 'string' && result._type.includes('QDM::')) {
-    // If there isn't a description, use the type name as a fallback.  This mirrors the frontend where we do
-    // result.constructor.name.
-    const description = result.description ? `${result.description}\n` : `${result._type.replace('QDM::', '')}\n`;
-    let startDateTime = null;
-    let endDateTime = null;
-    let startTimeString = '';
-    let endTimeString = '';
-    // Start time of data element is start of relevant period, if data element does not have relevant period, use authorDatetime
-    if (result.relevantPeriod) {
-      if (result.relevantPeriod.low) {
-        startDateTime = result.relevantPeriod.low;
-      }
-      if (result.relevantPeriod.high) {
-        endDateTime = result.relevantPeriod.high;
-      }
-    } else if (result.prevalencePeriod) {
-      if (result.prevalencePeriod.low) {
-        startDateTime = result.prevalencePeriod.low;
-      }
-      if (result.prevalencePeriod.high) {
-        endDateTime = result.prevalencePeriod.high;
-      }
-    } else if (result.authorDatetime) {
-      // TODO: start result string will need to be updated to AUTHORED once bonnie frontend
-      // updates its pretty printer to do so.
-      startDateTime = result.authorDatetime;
-    }
-
-    if (startDateTime) {
-      startTimeString = `START: ${moment.utc(startDateTime.toString()).format('MM/DD/YYYY h:mm A')}\n`;
-    }
-    // If endTime is the infinity dateTime, clear it out because we do not want to export it
-    if (endDateTime && endDateTime.year !== 9999) {
-      endTimeString = `STOP: ${moment.utc(endDateTime.toString()).format('MM/DD/YYYY h:mm A')}\n`;
-    }
-    const system = result.dataElementCodes[0].system;
-    // TODO: Sort out a better way to have a friendly system display for FHIR codes
-    const codeDisplay =
-      result.dataElementCodes && result.dataElementCodes[0] ? `CODE: ${system} ${result.dataElementCodes[0].code}` : '';
-    // Add indentation
-    const returnString = `${description}${startTimeString}${endTimeString}${codeDisplay}`;
-    return returnString.replace(/\n/g, `\n${currentIndentation}${keyIndentation}`);
-  } else if (result instanceof String || typeof result === 'string') {
+  } else if (typeof result === 'string') {
     return `"${result}"`;
-  } else if (result instanceof Array) {
-    prettyResultReturn = result.map((value: any) => prettyResult(value, indentLevel, keyIndent));
+  } else if (Array.isArray(result)) {
+    prettyResultReturn = result.map((value: any) => prettyResult(value, includeType, indentLevel, keyIndent));
     return `[${prettyResultReturn.join(`,\n${currentIndentation}${keyIndentation}`)}]`;
-  } else if (result instanceof Object) {
+  } else if (typeof result === 'object') {
```

Comment:
Omg. Today I learned that `typeof null === 'object'` which makes me very sad. Let's add another guard I guess. This throws an error if a test patient has an Encounter or something that is missing the queried for info in the kitchen sink example (like an encounter without a period)

```suggestion
  } else if (result != null && typeof result === 'object') {
```